test/00_setup.js

```diff
@@ -0,0 +1,23 @@
+const chai = require('chai');
+chai.use(require('chai-as-promised'));
+chai.use(require('dirty-chai'));
+
+const finalhandler = require('finalhandler');
+const http = require('http');
+const path = require('path');
+const serveStatic = require('serve-static');
+const serve = serveStatic(path.resolve(__dirname, 'fixtures'));
+const server = http.createServer(function onRequest (req, res) {
+  serve(req, res, finalhandler(req, res));
+});
+
+before(async function () {
+  const { address, port } = await new Promise((resolve, reject) => {
+    server.listen(0, 'localhost', () => resolve(server.address()));
+  });
+  this.host = `http://${address}:${port}`;
+});
```

Comment:
I guess this is fine, I'd generally be inclined just to `nock` these out.  Then each test is simple sth like:
```js
nock('https://thing/blerg.opus').replyWithFile(200, path.join(__dirname, 'blerg.opus'));
```

Reply:
But either way... this is more realistic I guess.

Reply:
@timlesallen the main reason for doing this is that nock does not work for ffprobe calls... I need an actual hosted url to test that properly.